lib/pronto/formatter/text_formatter.rb

```diff
@@ -1,44 +1,53 @@
 module Pronto
   module Formatter
-    class TextFormatter
-      include Colorizable
+    class TextFormatter < Base
+      class TextMessageDecorator < SimpleDelegator
```

Comment:
I nested it just because the decorator was only going to be used in this one place so I thought keeping it close improved readability. I don't feel strongly about it though so I'll reorganize it.